source/iface/param_traits.rst

```diff
@@ -2,23 +2,189 @@
   Copyright 2020 The Khronos Group Inc.
   SPDX-License-Identifier: CC-BY-4.0
 
-================
-``param_traits``
-================
+************************
+Backend interoperability
+************************
+
```

Comment:
I think it would be good to start the section with this paragraph which starts the related section within the SYCL spec.  Can we add this into the reference?

> Many of the SYCL runtime classes may be implemented such that they encapsulate an object unique to the SYCL backend that underpins the functionality of that class. Where appropriate, these classes may provide an interface for interoperating between the SYCL runtime object and the native backend object in order to support interoperability within an application between SYCL and the associated SYCL backend API.

Reply:
Added with commit https://github.com/KhronosGroup/SYCL_Reference/pull/93/commits/4a53bb4c0549314a5cc9dd1e8e76cca1e2c463fd